packages/ui/src/loading/LoadingSpinner.tsx

```diff
@@ -10,7 +10,11 @@ const LoadingSpinner: FunctionComponent<LoadingSpinnerProps> = ({ isLoading }) =
     }
 
     return (
-        <div className="loadingSpinner loadingOverlay-container" style={{ height: 100 }}>
+        <div
+            className="loadingSpinner loadingOverlay-container"
+            data-test="loading-spinner"
```

Comment:
Instead of utilizing test-id, can we create a user accessibility role and use it for testing?

Reply:
The rest looks good to me.

Reply:
Yeah, I've tried to do that but it's not comfortable to work with it in that case.
We have a loading spinner and have loading overlay(parent element) and I don't see any variants how to rearrange roles in that case. 
So, I'll add `role="status"` to spinner element, and will leave LoadingOverlay element with test-id.
If you have better ideas, please share

Reply:
Thank you.
I think `role="status"` is good as we don't have any other status role component, then it can be asserted by `screen.getByRole('status');`.